pkgs/tools/networking/cloudflare-warp/default.nix

```
@@ -12,11 +12,11 @@
 
 stdenv.mkDerivation rec {
   pname = "cloudflare-warp";
-  version = "2023.3.398";
+  version = "2023.3.470";
 
   src = fetchurl {
-    url = "https://pkg.cloudflareclient.com/uploads/cloudflare_warp_2023_3_398_1_amd64_002e48d521.deb";
-    hash = "sha256-1var+/G3WwICRLXsMHke277tmPYRPFW8Yf9b1Ex9OmU=";
+    url = "https://pkg.cloudflareclient.com/pool/jammy/main/c/cloudflare-warp/cloudflare-warp_2023.3.470-1_amd64.deb";
+    sha256 = "0189e68ac1102858840768912627c4a916ddcc0c8171fad4d084de52890a2da8";
```

Please use the SRI hash. This came from `nix hash to-sri --type sha256 0189e68ac1102858840768912627c4a916ddcc0c8171fad4d084de52890a2da8`
```suggestion
    hash = "sha256-AYnmisEQKFiEB2iRJifEqRbdzAyBcfrU0ITeUokKLag=";
```